realm/realm-annotations-processor/src/test/resources/io/realm/AllTypesRealmProxy.java

```diff
@@ -170,6 +171,9 @@ private void injectObjectContext() {
             if (!proxyState.getAcceptDefaultValue$realm()) {
                 return;
             }
+            final Row row = proxyState.getRow$realm();
+            row.getTable().setLong(columnInfo.columnLongIndex, row.getIndex(), value, true);
```

Comment:
Core provides SetDefault features only on `Table` class and I couldn't find a way to get `Table` instance from `Row` in JNI layer. `Row` seems to have the pointer to the `Table` as a private member and we can use it by modifying the Core.

Should I ask core team to expose SetDefault feature to `Row` class?